libvast/vast/system/node_control.hpp

```diff
@@ -55,4 +57,23 @@ get_node_components(caf::scoped_actor& self, caf::actor node,
   return result;
 }
 
+/// Look up a node component by component type and label
```

Comment:
Could you document the function fully? I.e., `@param` and `@returns` as well?

Reply:
I'd suggest we discuss first whether we really want to establish this as a standard practice. For user-facing API functions I'd definitely say yes, as well as for functions doing obscure things. 

However, there's some balance to consider, since I'd also like to avoid adding trivial comments like

```
    @param node The node containing the component
    @param type The type of the component
    @param label The label of the component
```

all over our codebase.

Reply:
Good point. We'll discuss the "documentation style" in a separate context.